doc/py_tutorials/py_setup/py_setup_in_ubuntu/py_setup_in_ubuntu.markdown

```diff
@@ -62,17 +62,37 @@ We need **CMake** to configure the installation, **GCC** for compilation, **Pyth
 
 ```
 sudo apt-get install cmake
-sudo apt-get install python-dev python-numpy
 sudo apt-get install gcc g++
 ```
 
+to support python2:
+
+```
+sudo apt-get install python-dev python-numpy
+```
+
+to support python3:
+
+```
+sudo apt-get install python3-dev python3-numpy
```

Comment:
BTW, `numpy` can be installed via `virtualenv` too (if no access to sudo, but other tools like gcc/cmake/python are here)